manage.py

```diff
@@ -367,6 +370,93 @@ def index_advisory_opinions():
                 print("%d docs loaded" % docs_loaded)
         print("%d docs loaded" % docs_loaded)
 
+def get_xml_tree_from_url(url):
+    r = requests.get(url, stream=True)
+
+    with open('temp.zip', 'wb') as f:
```

Comment:
I think it would be better to dump this in a temp file/dir with [mkstemp()/mkdtemp()](https://docs.python.org/3.5/library/tempfile.html?highlight=mktemp#tempfile.mkstemp)  for best security practices and potentially to avoid writing to disk (tmp space is probably an in-memory file system).